lib/SILOptimizer/Transforms/CopyForwarding.cpp

```diff
@@ -1645,7 +1645,8 @@ bool TempRValueOptPass::collectLoads(
                             << *user);
     return false;
 
-  case SILInstructionKind::ApplyInst: {
+  case SILInstructionKind::ApplyInst:
+  case SILInstructionKind::TryApplyInst: {
     ApplySite apply(user);
```

Comment:
Can this ever work if something isn't a full apply site? If not, can you make this a full apply site?

Reply:
Yeah, that would probably be more descriptive. But using an ApplySite does not hurt here, because the switch case already selects only full apply sites: ApplyInst and TryApplyInst.